google-cloud-pubsub/pom.xml

```diff
@@ -41,7 +41,10 @@
       <groupId>com.google.api.grpc</groupId>
       <artifactId>proto-google-common-protos</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-oauth2-http</artifactId>
+    </dependency>
```

Comment:
This is my addition and it fails somehow. https://github.com/googleapis/java-pubsub/pull/882#issuecomment-946275722